### DISCUSSION TITLE
Are docs updated?  Need new path to replace \rias\scout\

### DISCUSSION THREAD

**Discussion by npegman** [studioespresso/craft-scout#305] [General]
Am I missing something in that the install no longer creates a 'rias' dir in vendors?  What is the correct, updated path?
\studioespresso\craft-scout\ScoutIndex::create('scout_test')  doesn't appear to work.  Thanks for any input, bit of a noob with working in this fashion!

**Comment by janhenckens** [comment]
The plugin changed owner of couple of years back but to not break people's configs we decided to not change the namespace. Regardless of where the file is in composer you should still be able to use `\rias\scout\ScoutIndex::create()`.

Using the reference from the docs should do the trick :)

**Comment by npegman** [comment]
Ahh!  The namespace thing is new to me.  Thanks for the clarification and the great plugin.

**Comment by npegman** [comment]
``One more question if I may,  I've installed the Scout plugin (first via plugin manager and again via CLI) into Craft 5.  I then went to Algolia and setup an index and then setup scout.php in my config directory (see attached), moved my keys to env, verified that my index shows up under utilities/Scout Indices (shows 2 elements in Craft and 0 Records), but no Scout commands are working (i.e. - ddev craft scout/index/import 'my_index').  All commands fail to run with exit_status 1.  The logs on Algolia don't seem helpful.  They only show error status codes 200.

Craft logs under Queue Manager are showing 'json_encode error: Recursion Detected' and Job Data:

{
    "description": null,
    "indexName": "my_index"
}

Wondering what I should look at next?  Uninstall and reinstall Scout?   Appreciate any help!

scout.php: 
[scoutphp.txt](https://github.com/user-attachments/files/15842393/scoutphp.txt)

**Reply by janhenckens** [reply]
You can try disabling the `queue` setting and then saving an entry, that should show you a similar error, which you can then share here.
